### PR TITLE
[IMP]qweb:solve t-field integer issue while render on frontend.

### PR DESCRIPTION
**Pull request by vidhin6**
TASK:
https://www.odoo.com/web#id=10730&view_type=form&model=project.task&action=333&active_id=131
PAD:
https://pad.odoo.com/p/r.CCvjCUjYWuJdpnIw

### CODE REVIEW

File: openerp/addons/base/ir/ir_qweb.py [hunk 1]
```diff
@@ -650,6 +650,18 @@ def user_lang(self, cr, uid, context):
...
         return Lang.browse(cr, uid, lang_ids[0], context=context)
 
+class IntegerConverter(osv.AbstractModel):
+    _name = 'ir.qweb.field.integer'
+    _inherit = 'ir.qweb.field'
+
+    def value_to_html(self, cr, uid, value, field, options=None, context=None):
+        if context is None:
+            context = {}
+        fmt = '%d'
```
Comment (xmo-odoo) [inline openerp/addons/base/ir/ir_qweb.py]: Why have the format string as a separate variable?

**Comment by rim-odoo** [comment]
@Gorash Can I have your opinion on this PR ?

**Comment by Gorash** [comment]
All this looks good

File: openerp/addons/base/ir/ir_qweb.py (hunk 1 above)
Comment (rim-odoo) [inline openerp/addons/base/ir/ir_qweb.py]: @apa-odoo Please fix

Comment (apa-odoo) [inline openerp/addons/base/ir/ir_qweb.py]: Done

**Pull request closed without merging**